src/rust/engine/src/rule_graph.rs

```diff
@@ -602,6 +690,54 @@ impl RuleGraph {
   }
 }
 
+impl fmt::Display for RuleGraph {
```

Comment:
Yeah. I agree. I was planning on taking this out, but with the support rotation stuff---I wanted to start the CI runs before fixing it.